homework/index.js

```diff
@@ -71,19 +77,26 @@ const calculateAverageRating = () => {
   reviewList.forEach((review) => {
     ratingSum += parseInt(review.rating);
   });
-  const averageRating = (ratingSum / reviewList.length).toFixed(2);
-
   const avgReviewRating = document.getElementById("avgReviewRating");
-  avgReviewRating.innerHTML = averageRating + "/5";
-
+  if (!reviewList.length) {
+    avgReviewRating.innerHTML = "No reviews";
+  } else {
+    const averageRating = (ratingSum / reviewList.length).toFixed(2);
+    avgReviewRating.innerHTML = averageRating + "/5";
+  }
   return averageRating;
 };
 
-const deleteReview = (review) => {
+const deleteReview = (message, rating) => {
   reviewList = reviewList.filter((r) => {
-    return review !== r;
+    return message !== r.message || rating !== r.rating;
+  });
+  const reviewCards = document.querySelectorAll(".reviewCard");
+  reviewCards.forEach((card) => {
+    if (card.innerHTML.includes(message) && card.innerHTML.includes(rating)) {
+      card.remove();
+    }
```

Comment:
Brisanje se ne bi smjelo radit prema sadržaju, uvijek prema nekom ID-ju.
Što znači da bi kod newReview-a trebalo postavljati i id property (smisliti kako ga najbolje postaviti, a da nikad ne dođemo u situaciju da imamo dva itema s istim id-jevima).

Ovo je više za ubuduće, sad ovo možemo mergeati. Good job 👍